katacoda/scenarios/app-developer-guide/using-cache-image.md

```diff
@@ -75,4 +75,5 @@ pack build localhost:5000/buildpack-examples/second-cache-image-example \
     --publish
 ```{{execute}}
 
-[samples]: https://github.com/buildpack/samples
\ No newline at end of file
+[samples]: https://github.com/buildpack/samples
+---
```

Comment:
Why is it adding these additional `---` at the end?

Reply:
I've cleaned this up now.  It was working around a rendering issue on katacoda.  Issue no-longer appears.